lib/teaspoon/environment.rb

```diff
@@ -26,12 +26,16 @@ def self.check_env!(override = nil)
 
     private
 
+    def self.find_root_dir
```

Comment:
this hound warning doesn't seem to be the convention used throughout teaspoon... will ignore for now ;)